File: org.conqat.engine.sourcecode/test-data/org.conqat.engine.sourcecode.shallowparser/python/loops.py
while a < n:
	print a
	
for i in xrange(1, 10, 2):
	print i
	
foo = [1, 2, 3, 3]
for b in foo:
	print b
	for x in range(5, 10):
		print x
		
for x in [x for x in range(1, 10)]:
	print x
	
for ship in self.ships[:]:
	ship.remove()

File: org.conqat.engine.sourcecode/test-data/org.conqat.engine.sourcecode.coverage.volume/loops.java
public class Foo {

	public static void main(String[] args) {

		for (int i = 0; i < 15; ++i) 
		{
			System.out.println();
		}

		do {
			System.err.println();
		} while (args.length < 1);

		while (args.length < 3) {
			System.err.println();
		}
	}
}